src/main.css

```diff
@@ -186,6 +194,11 @@ a {
     font-weight: bold;
 }
 
+.tab-picker > .tab > .age {
```

Comment:
I wonder if this should actually be even smaller. It looks a little cramped with just the text NEW in a bubble.

Reply:
Yeah agreed. I'll try a little more messing around to see what looks good